src/main/java/uk/gov/justice/hmpps/nomis/datacompliance/repository/jpa/repository/DuplicateOffenderRepository.java

```diff
@@ -18,17 +17,17 @@
      * the first part (year) of the number.
      */
     @Query(value =
-                    "SELECT O1.OFFENDER_ID_DISPLAY FROM OFFENDERS O1 " +
+                    "SELECT DISTINCT(O1.OFFENDER_ID_DISPLAY) FROM OFFENDERS O1 " +
                     "INNER JOIN OFFENDER_IDENTIFIERS OI1 " +
                     "ON O1.OFFENDER_ID = OI1.OFFENDER_ID " +
-                    "AND OI1.IDENTIFIER_TYPE = 'PNC'" +
-                    "AND REGEXP_SUBSTR(UPPER(OI1.IDENTIFIER), '[0-9]{2,2}/[0-9]+[A-Z]+') IN (" +
-                    "    SELECT REGEXP_SUBSTR(UPPER(OI2.IDENTIFIER), '[0-9]{2,2}/[0-9]+[A-Z]+') FROM OFFENDERS O2" +
+                    "AND OI1.IDENTIFIER_TYPE = 'PNC' " +
+                    "AND CONCAT(REGEXP_SUBSTR(TRIM(UPPER(OI1.IDENTIFIER)), '[0-9]{2,2}/'), REGEXP_SUBSTR(TRIM(UPPER(OI1.IDENTIFIER)), '[1-9][0-9]*[A-Z]+$')) IN (" +
```

Comment:
Basically this now compares (the two digits before the slash) + (the characters after the slash minus leading zeros)